packages/versioning/src/validate.ts

```diff
@@ -799,8 +798,7 @@ function validateAvailabilityForRef(
   targetAvail: Map<string, Availability>,
   source: Type,
   target: Type,
-  sourceOptions?: TypeNameOptions,
```

Comment:
those two parameters are never really passed in, so I delete them for simplicity